src/tests/test_cases/test.py

```diff
@@ -53,6 +54,23 @@ def nested():
         ''
 
 
+@overload
+def overloaded_func(a: int) -> str:
+    ...
+
+
+@overload
+def overloaded_func(a: str) -> str:
+    # TODO(Find a way to test D418
```

Comment:
Can we test this somehow?

Maybe add an integration test and see if it raises the right error?

Reply:
See https://github.com/PyCQA/pydocstyle/blob/master/src/tests/test_integration.py

Reply:
Looks like we can :)